sqrl-tools/sqrl-packager/src/main/java/com/datasqrl/packager/Packager.java

```diff
@@ -294,7 +293,7 @@ public static Optional<List<Path>> findPackageFile(Path rootDir, List<Path> pack
     if (packageFiles.isEmpty()) {
       Path defaultPkg = rootDir.resolve(DEFAULT_PACKAGE);
       if (Files.isRegularFile(defaultPkg)) {
-        return Optional.of(List.of(defaultPkg));
+        return Optional.of(Lists.newArrayList(defaultPkg));
```

Comment:
It's probably a safer assumption to assume that any list being passed between functions is immutable or assumed to be immutable. We should just create a mutable list and populate it at the point of mutation.